content/_data/changelogs/weekly.yml

```diff
@@ -10730,6 +10730,82 @@
     authors:
       - daniel-beck
 
+- version: '2.288'
+  date: 2021-04-11
+  changes:
+  - type: major bug
+    category: regression
+    pull: 5408
+    issue: 65336
+    authors:
+    - MarkEWaite
+    message: |-
+      Fix load statistics graph links to include correct graph duration (regression in 2.264).
+  - type: rfe
+    category: rfe
+    pull: 5402
+    issue: 65308
+    authors:
+    - res0nance
+    message: |-
+      Reduce queue-lock contention when creating or deleting nodes.
+  - type: rfe
+    category: rfe
+    pull: 5392
+    authors:
+    - uhafner
+    message: |-
+      Simplify icon for unstable builds (use an exclamation mark).
+  - type: bug
+    category: bug
+    pull: 5377
+    issue: 56934
+    authors:
+    - jtnord
+    message: |-
+      Honor the current folder when creating new views with the "New View" link.
+  - type: rfe
+    category: localization
+    pull: 5387
+    authors:
+    - mustafau
+    message: |-
+      Add Turkish translations for new login form.
+  - type: rfe
+    category: developer
+    pull: 5385
+    authors:
+    - alecharp
+    message: |-
+      Developer: Remove unnecessary bundled handlebars asset.
+  - type: rfe
+    category: developer
+    pull: 5391
+    issue: 65288
+    authors:
+    - Wadeck
+    message: |-
+      Developer: <code>stapler.jelly.trace</code> support is back.
+  - type: rfe
+    category: internal
+    pull: 5324
+    authors:
+    - daniel-beck
+    - jglick
+    message: |-
+      Internal: Update Stapler to 1.263 to make the commons-beanutils update work.
```

Comment:
Reference changelog in https://github.com/stapler/stapler/releases/tag/stapler-parent-1.263 ?
Also "to make the commons-beanutils update work"  is a bit confusing, because it was not broken

Reply:
Very good catch.  I've updated the changelog entry to include a reference to the Stapler 1.263 changelog and the Apache commons-beanutils 1.9.4 changelog, since they were both updated in that pull request.  I've added a sentence that notes the upgrade of commons-beanutils so that we're consistently stating those updates in the changelog.

See 52fe850f617863c7bc69386893e9621e921cc7ff for the change